src/index.spec.ts

```diff
@@ -256,6 +257,36 @@ describe('EppoJSClient E2E test', () => {
     });
   });
 
+  describe('LocalStorageAssignmentCache', () => {
```

Comment:
testing the cache directly instead of as an integration with `getAssignment` as those already existing in the commons